src/main/resources/swagger.api/schemas/holding.json

```diff
@@ -82,5 +82,6 @@
     "metadata": {
       "$ref": "metadata.json"
     }
-  }
+  },
+  "required": ["electronicAccess", "notes"]
```

Comment:
More strict schema? what has it got to do with this PR?

Reply:
This is explained in the "Implementation details" section of the pull request description.

Reply:
If you want the client that calls mod-search to make the normalization I can remove the normalization in mod-search, and implement a hamcrest matcher that ignores these differences in the unit tests.